gef.py

```diff
@@ -1528,13 +1532,13 @@ def titlify(text, color=None, msg_color=None):
     return "".join(msg)
 
 
-def err(msg):   return gef_print("{} {}".format(Color.colorify("[!]", "bold red"), msg))
-def warn(msg):  return gef_print("{} {}".format(Color.colorify("[*]", "bold yellow"), msg))
-def ok(msg):    return gef_print("{} {}".format(Color.colorify("[+]", "bold green"), msg))
-def info(msg):  return gef_print("{} {}".format(Color.colorify("[+]", "bold blue"), msg))
+def err(msg: str) -> Optional[int]:   return gef_print("{} {}".format(Color.colorify("[!]", "bold red"), msg))
```

Comment:
these don't need to be one liners anymore. Too hard to read with the annotations.